libr/util/graph.c

```diff
@@ -12,7 +12,8 @@ static RGraphNode *r_graph_node_new(void *data) {
 	RGraphNode *p = R_NEW0 (RGraphNode);
 	if (p) {
 		p->data = data;
-		p->free = NULL;
+// R_NEW0 already sets p->free to NULL
```

Comment:
```suggestion
```

Reply:
comment is so that nobody who reads this code reports a bug, where there is none